src/components/cylc/workflow/Toolbar.vue

```diff
@@ -77,9 +80,34 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           :disabled="!enabled.stopToggle"
           @click="onClickStop"
           size="small"
+          density="comfortable"
         />
       </div>
 
+      <!-- n-window selector -->
+      <v-chip
```

Comment:
Also how about
```suggestion
      <v-chip
        :disabled="isStopped"
```
or even

```suggestion
      <v-chip
        v-if="!isStopped"
```